tensorboard/plugins/projector/vz_projector/vz-projector-inspector-panel.ts

```diff
@@ -282,12 +327,35 @@ namespace vz_projector {
       this.searchBox.message = '';
       const minDist = neighbors.length > 0 ? neighbors[0].dist : 0;
 
+      const spriteImagePath = this.spriteMeta.imagePath;
+      if (spriteImagePath) {
+        var {aspectRatio, nCols} = this.spriteMeta;
+        var paddingBottom = 100 / aspectRatio + '%';
+        var backgroundSize = `${nCols * 100}% ${nCols * 100}%`;
+      }
+
       for (let i = 0; i < neighbors.length; i++) {
         const neighbor = neighbors[i];
 
         const neighborElement = document.createElement('div');
         neighborElement.className = 'neighbor';
 
+        if (spriteImagePath) {
+          var neighborElementImage = document.createElement('div');
+          neighborElementImage.className = 'neighbor-image';
+          neighborElementImage.style.backgroundImage = `url(${spriteImagePath})`;
```

Comment:
Can we escape this URL with `CSS.escape(spriteImagePath)`?

At this point, might as well precompute the whole thing outside the loop:

```javascript
const backgroundImage = `url(${CSS.escape(this.spriteMeta.imagePath)})`;
```